### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by jkauffman503**
Minor updates to README (formatting, consistency, more instructions),
and adding Contributing.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,71 @@
...
+# Netuitive Dropwizard Integration
+Dropwizard is part Java framework and part Java library that assists in operating web services. The Netuitive Dropwizard integration uses a [DropWizard.io](http://www.dropwizard.io/) `netuitive` reporter to send metrics to [Netuitive's Linux agent](https://help.netuitive.com/Content/Integrations/linux.htm).
+
+## Using the DropWizard Integration
+
+1. Include the appropriate Ananke library dependency. You'll also need a working StatsD ([Netuitive StatsD](https://help.netuitive.com/Content/Integrations/netuitive_statsd.htm) or [Etsy StatsD](https://help.netuitive.com/Content/Integrations/statsd.htm)) integration.
+1. Add `dropwizard-metrics` to your POM.
+
+        <dependency>
+          <groupId>io.dropwizard.metrics</groupId>
```
Comment (TheConnMan) [inline README.md]: @joepusateri Does this group need to be updated to `com.netuitive`?

**Review by TheConnMan — COMMENTED** [review]
LG2M, just the one question for Joe.

File: README.md (hunk 1 above)
Comment (joepusateri) [reply README.md]: Yes, good catch!

**Comment by jkauffman503** [comment]
@TheConnMan @joepusateri readme updated

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**